src/wp-admin/includes/class-plugin-upgrader.php

```diff
@@ -262,7 +262,7 @@ public function upgrade( $plugin, $args = array() ) {
 
 		if ( isset( $past_failure_emails[ $plugin ] ) ) {
 			unset( $past_failure_emails[ $plugin ] );
-			update_option( 'auto_plugin_theme_update_emails', $past_failure_emails );
+			update_option( 'auto_plugin_theme_update_emails', $past_failure_emails, false );
```

Comment:
I don't think we needs to change it, In https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/includes/schema.php#L578C3-L613 core will set `off`.